pagopa/io-app
# fix(IT Wallet): [SIW-1724] Add code 12 in skeumorphic mdl card if present

<img width="432" alt="Screenshot 2024-10-09 at 17 39 13" src="https://github.com/user-attachments/assets/160fe5f6-7db7-4965-ba59-33f319d0c3bb">
<img width="432" alt="Screenshot 2024-10-09 at 17 39 22" src="https://github.com/user-attachments/assets/70e60e13-6eb2-439b-8f37-65ba23be69f7">



## Short description
This PR inserts the value of the code, if any, in column 12 of the skeumorphic mdl card. Previously, it was only possible to see the code in the details of a specific driving license ( e.g., driving license B) in the driving license claims 

## List of changes proposed in this pull request
The mdl mock has been updated to include the restricted conditions for a specific driving license
A new CardClaimContainer has been added to the to display the value of the restricted conditions in column 12

## How to test
Use the mocked mdl credential and see if in column 12 of the type B license the value is 78
